app/constants/sso.ts

```diff
@@ -3,6 +3,9 @@
 
 import keyMirror from '@utils/key_mirror';
 
+export const REDIRECT_URL_SCHEME = 'mmauth://';
+export const REDIRECT_URL_SCHEME_DEV = 'mmauthbeta://';
```

Comment:
@anurag6713 just FYI that we are adding the hardcoded schemes for authentication here.. 

This is for **Gekidou** so once you make the change for having these values dynamically set, make sure the changes are also applied to the *gekidou* branch

Reply:
Sure @enahum. Thank you.